core/controllers/base_test.py

```diff
@@ -1174,7 +1039,32 @@ def test_every_method_has_decorator(self):
         self.assertGreater(len(handlers_checked), 0)
 
         for (name, method, handler_is_decorated) in handlers_checked:
-            self.assertTrue(handler_is_decorated)
+            with self.subTest('%s.%s' % (name, method)):
+                self.assertTrue(handler_is_decorated)
+
+
+class MainTests(test_utils.GenericTestBase):
```

Comment:
It is not clear what this is testing -- I don't see an assertion except in the helper functions (whose names should also start with verbs). Maybe you can pick a clearer test name and leave line comments to explain what's being tested here exactly.

Reply:
I moved this test to main_test.py and also refactored it a bit.